### PR TITLE
feat(graph): Add move all selected on drag

### LINKED ISSUE #3770: Graph2 needs ability to move all selected items on drag

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes #3770

Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [ ] The commit is signed.
- [x] The change has been fully tested:
  - [x] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: packages/graph/src/graph2/graph.css [hunk 1]
```diff
@@ -1,9 +1,11 @@
 .graph_Graph2 .graphVertex {
     cursor: pointer;
+    pointer-events: all;
```
Comment (GordonSmith) [inline packages/graph/src/graph2/graph.css:3]: Is this needed?

**Review by GordonSmith — COMMENTED** [review]
One comment

File: packages/graph/src/graph2/graph.css (hunk 1 above)
Comment (jbrundage) [reply packages/graph/src/graph2/graph.css:3]: I don't think so... checking...

Comment (jbrundage) [reply packages/graph/src/graph2/graph.css]: Seems to be fine without it.

**Pull request merged**